cmd/dep/signals.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
Yeah, I was sloppy with these bits. OK, so...

`Release()` is something that should be promoted into the `SourceManager` interface. I'm fine with the idea that any `SourceManager` should necessarily have a lifecycle. There's no way they can do the job promised by the interface _without_ doing state management (unless they're just mocks), so there's no sense in omitting lifecycle methods from the interface.

`HandleSignals` is something I'm still shifting on, though. Per #160, having the library set up and do the signal handling itself isn't exactly ideal, even if it is triggered by the caller.